api/app/Http/Controllers/UserController.php

```diff
@@ -2,15 +2,33 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\College;
+use App\Models\Course;
+use App\Models\Department;
 use App\Models\User;
+use Illuminate\Contracts\Database\Eloquent\Builder;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
 
 class UserController extends Controller
 {
-    //
+    public function getAuthUser()
+    {
+        $user = Auth::user();
+        $id = Auth::id();
```

Comment:
I think you can get the id from the `$user`

Reply:
Comment applied.